src/wp-includes/js/admin-bar.js

```diff
@@ -266,6 +417,15 @@ if ( typeof(jQuery) != 'undefined' ) {
 			}
 		},
 
+		/**
+		 * Handles the click on a shortlink in the adminbar.
+		 *
+		 * @since 3.5.0
+		 *
+		 * @param {HTMLElement} e The HTML element.
```

Comment:
Isn't this the event rather than the element? 
Note: this is already merged in core in https://core.trac.wordpress.org/changeset/43332
/Cc @atimmer

Reply:
Thanks, fixed it: https://core.trac.wordpress.org/changeset/43341.